### PR TITLE
using cron to schedule tasks

### PR DESCRIPTION
**Pull request by Ozgur2096**

### CODE REVIEW

File: server/src/controllers/user.js [hunk 1]
```diff
@@ -317,16 +321,23 @@ export const getProfile = async (request, response) => {
...
         } else {
-          response.status(200).json({
-            success: true,
-            user: info,
-          });
+          const user = await User.findById(info.id);
+          if (!user) {
+            response
+              .status(200)
+              .json({ success: true, message: 'User not found' });
```
Comment (sergeyzoloto) [inline server/src/controllers/user.js:334]: Some clarification is required here. I don't quite understand why if a user is not found, the server responds with status 200 and success: true

File: server/src/app.js [hunk 2]
```diff
@@ -23,8 +25,15 @@ app.use(cors(corsOptions));
 
 // Cookies
 import cookieParser from 'cookie-parser';
+import { deleteUserIfExpired } from './models/User.js';
 app.use(cookieParser());
 
+// We use cron to schedule tasks
+cron.schedule('00 * * * *', () => {
+  // running a task every hour
```
Comment (sergeyzoloto) [inline server/src/app.js]: How can the regularity of the scan be reconfigured? Say, reduce it from an hour to a day.

**Review by sergeyzoloto — CHANGES_REQUESTED** [review]
Just a small reminder. Please don't forget about the descriptive commit message. Start your commit message by saying that you added, removed, or changed something. Then write what the result is. By reading a commit message, an outsider can understand what was done and why

File: server/src/app.js (hunk 2 above)
Comment (Ozgur2096) [reply server/src/app.js]: Sorry, I forgot to change it, I am going to change it

**Review by sergeyzoloto — APPROVED** [review]

**Pull request merged**